<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 221, 221);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>511</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>강원교육모두 Bold</family>
     <pointsize>48</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>사진을 선택해 주세요!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextBtn">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>30</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/next_btn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="selectPicture1Btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="selectPicture3Btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>470</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="selectPicture2Btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="selectPicture4Btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="picture1Check">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/check_btn.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="picture2Check">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>270</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/check_btn.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="picture3Check">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>540</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/check_btn.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="picture4Check">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>540</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/check_btn.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>nextBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>changePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>927</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>984</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectPicture1Btn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>selectPicture1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectPicture2Btn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>selectPicture2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>853</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>978</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectPicture3Btn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>selectPicture3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectPicture4Btn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>selectPicture4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>859</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>937</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changePage()</slot>
  <slot>selectPicture1()</slot>
  <slot>selectPicture2()</slot>
  <slot>selectPicture3()</slot>
  <slot>selectPicture4()</slot>
 </slots>
</ui>
